scripts/js/commands/checkKatexRender.test.ts

```diff
@@ -0,0 +1,62 @@
+// This code is a Qiskit project.
+//
+// (C) Copyright IBM 2025.
+//
+// This code is licensed under the Apache License, Version 2.0. You may
+// obtain a copy of this license in the LICENSE file in the root directory
+// of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+//
+// Any modifications or derivative works of this code must retain this
+// copyright notice, and modified files need to carry a notice indicating
+// that they have been altered from the originals.
+
+import { expect, test } from "@playwright/test";
+
+import { hasInlineMath, removeFileExtension } from "./checkKatexRender";
+
+test("hasInlineMath()", () => {
+  const md1 = `
```

Comment:
Nit that it'd help to name these more clearly. Maybe `baselineMd`, `mathBlock`, and `inlineMath`